### PR TITLE
Frontpage aesthetic tweaks

### PR DESCRIPTION
**Pull request by StevenClontz**
Did some poking around the front page to get it ready for re-launch. I thought it might be best to have contributing guidelines factored out to pi-base/contributing so that can be easily maintained without access to the full server repo. I added @jamesdabbs and myself to the page so folks know who we are, and credited South for their contribution re: my USAFDC grant. I also did a bit of styling of the jumbotron and navbar.

### CODE REVIEW

File: viewer/src/index.css [hunk 1]
```diff
@@ -13,4 +13,21 @@ body {
...
 .graphiql-container {
   min-height: 800px;
-}
\ No newline at end of file
+}
+
+.jumbotron.wide {
+  margin-top:-20px; 
+  background-color: #222;
+  background-image: url('/peanocurve.svg');
```
Comment (jamesdabbs) [inline viewer/src/index.css:21]: 😍

File: viewer/src/components/Layout.tsx [hunk 2]
```diff
@@ -46,6 +48,19 @@ class Layout extends React.PureComponent<Props> {
       <div>
         <Navbar />
 
+
+        { location.pathname === '/' ?
```
Comment (jamesdabbs) [inline viewer/src/components/Layout.tsx]: Why isn't this just part of `<Home/>`? In any case, I'd prefer we did our routing consistently with `<Route/>`s like https://github.com/pi-base/server/blob/ede6b2c331e2c75a7c7e9ce9a925cbf1e42cb84d/viewer/src/routes.tsx#L21

Comment (jamesdabbs) [reply viewer/src/components/Layout.tsx]: Oh! It's because you don't want it in the `.container`. :+1:

Comment (StevenClontz) [reply viewer/src/components/Layout.tsx]: Yeah, although I opened #5 to get the best of both worlds.

**Pull request merged**